.github/workflows/publish.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@cae68fefc6b5f367a0275617c9f83181ba54714f # v1.7.0
         with:
-          charts_dir: ./chart
+          charts_dir: .
```

Comment:
I think it needs to point to a directory that contains a sub-directory with a chart in it